client/client-multi/client-c/src/kaa/kaa_context.h

```diff
@@ -75,6 +75,10 @@ typedef struct kaa_channel_manager_t    kaa_channel_manager_t;
     typedef struct kaa_configuration_manager kaa_configuration_manager_t;
 #endif
 
+#ifndef KAA_NOTIFICATION_MANAGER_T
```

Comment:
Please, align all definitions in one way, e.g.
#ifndef KAA_STATUS_T
# define KAA_STATUS_T
    typedef struct kaa_status_t             kaa_status_t;
#endif